### PR TITLE
Aliases for various commands for convenience

### PR DESCRIPTION
**Pull request by WitherredAway**
Adds aliases for
- collect search
- tag forcedelete
- tag forceedit

### CODE REVIEW

File: cogs/collectors.py [hunk 1]
```diff
@@ -99,7 +99,7 @@ async def query_collectors(self, species):
                 continue
             yield user
 
-    @collect.command()
+    @collect.command(aliases=['s'])
```
Comment (oliver-ni) [inline cogs/collectors.py]:
```suggestion
    @collect.command(aliases=("s",))
```

Comment (oliver-ni) [reply cogs/collectors.py]: Use a tuple, and double quotes.

**Review by oliver-ni — CHANGES_REQUESTED** [review]

File: cogs/collectors.py (hunk 1 above)
Comment (WitherredAway) [reply cogs/collectors.py]: Done!

File: cogs/tags.py [hunk 2]
```diff
@@ -282,4 +282,4 @@ async def claim(self, ctx, *, name):
 
 
 def setup(bot):
-    bot.add_cog(Tags(bot))
+    bot.add_cog(Tags(bot))
```
Comment (oliver-ni) [inline cogs/tags.py]:
```suggestion
    bot.add_cog(Tags(bot))

```

**Review by oliver-ni — CHANGES_REQUESTED** [review]

**Review by oliver-ni — APPROVED** [review]

**Pull request merged**